src/main/java/com/sap/sgs/phosphor/fosstars/data/github/ScansForVulnerableDependencies.java

```diff
@@ -43,12 +45,12 @@ protected Feature supportedFeature() {
   @Override
   protected Value fetchValueFor(GitHubProject project) throws IOException {
     for (CachedSingleFeatureGitHubDataProvider provider : providers) {
-      Value value = provider.fetchValueFor(project);
-      if (!value.isUnknown() && Boolean.TRUE.equals(value.get())) {
+      Value<Boolean> value = provider.fetchValueFor(project);
+      if (value.orElse(FALSE)) {
```

Comment:
May be this entire thing can be written in one line
- Like `return SCANS_FOR_VULNERABLE_DEPENDENCIES.value(value.orElse(FALSE))` ?

Reply:
If so, then only the first data provider will be called which is wrong.